python/ray/tune/tests/test_tuner_restore.py

```diff
@@ -176,6 +191,14 @@ def test_tuner_restore_restart_errored(ray_start_4_cpus, tmpdir):
     tuner = Tuner.restore(
         str(tmpdir / "test_tuner_restore_restart_errored"), restart_errored=True
     )
+
+    # Check restored results
+    results = tuner.get_results()
```

Comment:
actually can we add a test to see if get_best_result can be called successfully?

Reply:
Added